### PR TITLE
refactor(tests): Use `codeBlock` tagged literal

### PR DESCRIPTION
**Pull request by zharinov**
## Changes

- Nice indenting of code fragments used in tests
- Indenting level infromation doesn't leak to the code fragment
- Describing code blocks near the place where it's being used is safer

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/util/github/graphql/cache-adapter.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
+import { DateTime, DurationLikeObject } from 'luxon';
```
Comment (rarkins) [inline lib/util/github/graphql/cache-adapter.ts:1]: Is this an accidentally added file?

Comment (zharinov) [reply lib/util/github/graphql/cache-adapter.ts]: Yes

**Review by viceice — DISMISSED** [review]

**Review by rarkins — DISMISSED** [review]

**Comment by viceice** [comment]
it seems we've some test issues[^1] in `mint` manager
image

[^1]: https://app.codecov.io/gh/renovatebot/renovate/pull/18916

File: lib/modules/manager/mint/extract.spec.ts [hunk 2]
```diff
@@ -93,7 +82,12 @@ describe('modules/manager/mint/extract', () => {
     });
 
     it('Mintfile Includes Commented Out', () => {
-      const res = extractPackageFile(includesCommentedOutMintFileContent);
+      const res = extractPackageFile(codeBlock`
+        SwiftGen/SwiftGen@6.6.1
```
Comment (viceice) [inline lib/modules/manager/mint/extract.spec.ts:86]:
```suggestion
        SwiftGen/SwiftGen@6.6.1

```

**Review by viceice — APPROVED** [review]

File: lib/modules/manager/mint/extract.spec.ts (hunk 2 above)
Comment (zharinov) [reply lib/modules/manager/mint/extract.spec.ts:86]: Thanks 👍

**Pull request merged**